ClientApp/viking-vault/src/ExchangeForm.tsx

```diff
@@ -203,19 +203,24 @@ class ExchangeForm extends React.Component<any, IExchangeFormState> {
         var exchangeRate = this.getCurrencyExchangeRate();
         var balance = this.calculateExchangedBalance(exchangeRate, this.state.toExchangeAmount);
         balance = this.extractBanksFeeFromBalance(balance);
-
         if(this.state.fromCurrency === this.state.toCurrency){
             this.setState({
                 openModal: true,
                 modalMessage: "The sell and buy currency must be different!"
             })
         }
-        else if(this.state.toExchangeAmount < 1) {
+        else if(this.state.toExchangeAmount <= 0) {
             this.setState({
                 openModal: true,
                 modalMessage:  "The amount of money to exchange must be a positive number!"
             })
         }
+        else if(this.state.toExchangeAmount > this.state.availableAmountFromCurrency) {
+            this.setState({
+                openModal: true,
+                modalMessage:  "Not enough funds to execute the exchange!"
```

Comment:
I think that perform should be used here instead of execute

Reply:
🔪 execute

Reply:
Done, changed it, thank you.